pkg/ddc/alluxio/ufs.go

```diff
@@ -90,6 +92,19 @@ func (e *AlluxioEngine) ShouldUpdateUFS() (ufsToUpdate *utils.UFSToUpdate) {
 	ufsToUpdate = utils.NewUFSToUpdate(dataset)
 	ufsToUpdate.AnalyzePathsDelta()
 
+	// 3. for hostpath ufs mount, check if all mountpoints have been mounted
+	const envVar = "FLUID_ENABLE_REMOUNT_DURING_SYNC"
+	if os.Getenv(envVar) != ""{
+		unmountedPaths, err := e.FindUnmountedUFS()
```

Comment:
**I have one question here:**
Since we check Alluxio for the unmounted UFS(e.g. `alluxio fs mount`) every sync loop, do we still need `Dataset.status.mounts` to indicate the currently mounted UFSs?

WDYT @yangyuliufeng @cheyang

Reply:
I agree with you. if we have to check `fs mount` in every loop, we don't need `Dataset.status.mounts`. The purpose of  `Dataset.status.mounts` is to make the `remount` or `mount` as less as possible.

Reply:
But my newly add bug fix is protected with Env environment and only targeted for non-hostpath scenario, not sure if this additional check is covering all scenarios removing mounts part will effect.